README.md

```diff
@@ -27,7 +27,7 @@ To get a local copy up and running follow these simple example steps.
 
```

Comment:
**[Optional]**

- [ ] As you know that this is a pair-programming but i cannot see any commit by @Javivy. It would be my humble request from @Javivy to make contributions in this project to follow the requirements 😊

Reply:
well resolved! ✔️